source/heroes/ScreensaverHero.hx

```diff
@@ -0,0 +1,47 @@
+package heroes;
+
+import flixel.FlxSprite;
+import flixel.util.FlxColor;
+import flixel.FlxG;
+
+class ScreensaverHero extends FlxSprite {
+    var DirectionX:Float;
+    var DirectionY:Float;
+
+    public function new(X:Float=0, Y:Float=0)
+    {
+        super(X, Y);
+        makeGraphic(20, 40, 0xFFFFFFFF);
```

Comment:
Same sort of comment as above, let's throw magic numbers into variables to help the logic stay clean and easier to follow.